packages/ui-components/src/components/enable-button.ts

```diff
@@ -0,0 +1,15 @@
+import { html } from 'lit';
+import { BaseElement } from '../base';
+
+export class EnableButton extends BaseElement {
+  render() {
+    return html`
+      <button
+        class="transition-all flex items-center flex-row align-center text-white border-2 border-neutral-600 rounded-md box-border font-bold"
+      >
+        <div class="z-[2] bg-neutral-600 box-border p-2">+</div>
+        <p class="p-2">Enable Repository</p>
+      </button>
+    `;
+  }
+}
```

Comment:
This component could be more generic, something like `ImageButton`.

Enable button could then make use of `ImageButton`. It could also be nice to add a directory for more generic components and a directory for less generic (more specific) components.

Reply:
https://bradfrost.com/blog/post/atomic-web-design/